app/lib/admin/actions/package_info.dart

```diff
@@ -27,11 +29,18 @@ Loads and displays the package information.
       throw NotFoundException.resource(package);
     }
 
+    final uploaderIds = p.uploaders;
+    List<User>? uploaders;
+    if (uploaderIds != null) {
+      uploaders =
+          (await accountBackend.lookupUsersById(uploaderIds)).nonNulls.toList();
```

Comment:
Note: `getEmailsOfUserIds` will use cache and may be faster.

Reply:
Nice, switched